Sources/Templates/Extensions/Method+Extension.swift

```diff
@@ -246,6 +263,28 @@ private extension Method {
         }
         return parts.compactMap { $0 }.joined(separator: " ")
     }
+
+    func functionName() -> String {
```

Comment:
Maybe name it `methodName()` to match the object type in Sourcery. And this could be a computed var? Same for below.